target/linux/ramips/image/mt7621.mk

```diff
@@ -145,6 +149,19 @@ define Device/asiarf_ap7621-nv1
 endef
 TARGET_DEVICES += asiarf_ap7621-nv1
 
+define Device/asus_rp-ac56
+  DEVICE_VENDOR := ASUS
+  DEVICE_MODEL := RP-AC56
+  IMAGE_SIZE := 16000k
+  BLOCKSIZE := 64k
+  DEVICE_PACKAGES := kmod-mt7603 kmod-mt76x2 wpad-basic kmod-i2c-core \
```

Comment:
kmod-i2c-core will be automatically selected by kmod-i2c-ralink, kmod-sound-core will be automatically selected by kmod-sound-mt7620. So, it should be okay to remove both.

Reply:
Done